main.go

```diff
@@ -136,7 +137,7 @@ func argsParse(argv []string) (args Args) {
   -h --help                   Show this screen.
   --version                   Output version and exit.
   --sources=<sources>         Comma separated list of feature sources.
-                              [Default: cpuid,rdt,pstate,memory,network,storage,selinux]
+                              [Default: cpuid,iommu,rdt,pstate,memory,network,storage,selinux]
```

Comment:
this list probably should be sorted alphabetically ?

Reply:
Yeah, I actually did this, but decided to move that into a separate patch. The order here is a leftover from that. There's no PR about that, yet.

Reply:
In the end, labels are represented as `map[string]string` and maps are unordered in Go. But I agree that we should sort the sources for readability.